### DISCUSSION TITLE
option 65 not set

### DISCUSSION THREAD

**Discussion by GinTonic99** [gerbera/gerbera#3287] [Q&A]
After an update to 2.3, gerbera would not start anymore with the following message:
`error: [/build/src/config/config_manager.cc:486] virtual std::string ConfigManager::getOption(ConfigVal) const Error: option 65 not set`
I did not find any hint in the changelog requiring modifications of the onfig file.
What is missing here in my config file? (the error message is not relly explicative or helpful)
br Gin

**Comment by KarlStraussberger** [comment]
This is a know issue. There is a simple workaround: https://github.com/gerbera/gerbera/issues/3230#issuecomment-2367477252

Hope that helps

**Reply by GinTonic99** [reply]
Many thanks for the super fast resonse, this worked. Actually I have ssen this thread but did not understand the connection to my issue.
thanks again and best regards gin
